### ISSUE TITLE
Unable to login neither with default user name and password not with specified via environment variables

### ISSUE THREAD

**Issue by artemkovalyov** [bitnami/bitnami-docker-magento#68]
On documentation page, only http(80) is being mapped while image redirects to https(443) for 

Here are logs after install: 
```
magento INFO  ########################################################################
magento INFO   Installation parameters for magento:
magento INFO     Username: artem
magento INFO     Site URL: http://127.0.0.1/
magento INFO     Admin URL: http://127.0.0.1/admin
magento INFO   (Passwords are not shown for security reasons)
magento INFO  ########################################################################
magento INFO
nami    INFO  magento successfully initialized
INFO  ==> Starting magento...
```

**Description**

**Steps to reproduce the issue:**

1. Run container
2. Go to admin page
3. Submit login from

**Describe the results you received:**

can't login, form is empty again

**Describe the results you expected:**

admin ui

**Additional information you deem important (e.g. issue happens only occasionally):**

also add https port mapping to your docs for manual install or it doesn't open login form at all in docker
**Version**

- Output of `docker version`: 

```
Docker version 1.12.6, build 78d1802
```

- Output of `docker info`:

```
(paste your output here)
```

- Output of `docker-compose version` (if applicable):

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

**Comment by carrodher** [comment]
Hi @artemkovalyov,
Really this is not an issue, it is the way Magento is configured. We provide everything necessary to use HTTPS, but it is an option that must be enabled from the Magento configuration through its web interface. To enable this option:

1. From the dashboard, go to `stores -> configuration`
2. Inside General section, go to `Web`
3. Once Web section is loaded, click on `Base URLs (Secure)` and choose **Yes** in `Use Secure URLs in Admin`
4. Click on `Save Config`
5. After that, you should follow the link in the warning message and click on `Flush Magento Cache`

![screen shot 2017-09-22 at 11 41 22](https://user-images.githubusercontent.com/13216600/30738771-6d686128-9f8b-11e7-8245-4fd0f158db37.png)

Apart from that, we will study the option to enable this option by default.

**Comment by marcosbc** [comment]
Hi all, we have just released a new major release of the Bitnami Docker image for Magento, [2.4.1-debian-10-r80](https://github.com/bitnami/bitnami-docker-magento#241-debian-10-r80). It includes major changes with respect to how the image is initialized, and now the source code for the initialization scripts is also published in the [rootfs](https://github.com/bitnami/bitnami-docker-magento/tree/master/2/debian-10/rootfs) folder of this repo.

As part of these changes, we added support for the `MAGENTO_ENABLE_HTTPS` and `MAGENTO_ENABLE_ADMIN_HTTPS` environment which let you decide where to enable HTTPS. By default it will be disabled in both store and admin respectively.
